NDcPP_v2_2e.adoc

```diff
@@ -1970,13 +1970,13 @@ A TOE may act as the client, the server, or both in TLS sessions. The requiremen
 
 _The use of X.509v3 certificates for TLS is addressed in FIA_X509_EXT.2.1. This requirement adds that the client must be capable of presenting a certificate to a TLS server for TLS mutual authentication._
 
-*FCS_TLSC_EXT.3 TLS Client Support for renegotiation (TLSv1.1/v1.2 only)*
+*FCS_TLSC_EXT.3 TLS Client Support for renegotiation (TLSv1.2 only)*
```

Comment:
For consistency, the above changes have all removed instances of the lowercase 'v' in the protocol version. Same should be done here (and forward of here).